src/fdc3/dotnet/DesktopAgent/DesktopAgent.sln

```diff
@@ -5,11 +5,11 @@ VisualStudioVersion = 17.4.33103.184
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{75D43F58-806B-4C28-BDA1-7A7AFF0250DB}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DesktopAgent", "src\DesktopAgent\DesktopAgent.csproj", "{4A71A18B-6DC1-44AD-85F3-200CC2CBC6BD}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MorganStanley.ComposeUI.DesktopAgent", "src\MorganStanley.ComposeUI.DesktopAgent\MorganStanley.ComposeUI.DesktopAgent.csproj", "{4A71A18B-6DC1-44AD-85F3-200CC2CBC6BD}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{44C08E9B-2A1D-4F01-B579-59DFCE4CBF6D}"
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "testt", "{44C08E9B-2A1D-4F01-B579-59DFCE4CBF6D}"
```

Comment:
testt?

Reply:
that happens when i stop wearing my glasses, I will fix that :)